mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -214,7 +217,7 @@ def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
                     logging.exception("Failed to create socket connection!")
                     raise
                 time.sleep(1)
-
+        self.port = port
```

Comment:
Ah didn't notice the 'addr' arg.
Do we need to have a default?
Perhaps it's better to force caller to be explicit about addr; also we can have the nice order of (addr, port, ...) as opposed to the reverse.